### PR TITLE
[Merged by Bors] - Fast sim

### PR DESCRIPTION
**Pull request by antonlerner**

### CODE REVIEW

**Comment by antonlerner** [comment]
bors try

File: activation/activation.go
Comment (y0sher) [inline activation/activation.go]: asd?

File: activation/activation.go
Comment (y0sher) [inline activation/activation.go]: asd?

File: activation/activation_test.go
Comment (y0sher) [inline activation/activation_test.go]: if you want to keep this please leave a comment why

File: cmd/node/app_test.go
Comment (y0sher) [inline cmd/node/app_test.go:118]: just a note, poet api might have changed. you should revisit it before merging

File: cmd/node/app_test.go
Comment (y0sher) [inline cmd/node/app_test.go]: comment is misleading no pod concept in here

File: cmd/node/multi_node.go
Comment (y0sher) [inline cmd/node/multi_node.go]: kinda funny to re-assign just for rephrasing haha

File: cmd/node/multi_node.go
Comment (y0sher) [inline cmd/node/multi_node.go]: why creating a tx is part of this func ? maybe take it out

File: cmd/node/multi_node.go
Comment (y0sher) [inline cmd/node/multi_node.go]: you can replace format date with `time.RFC3339`

File: cmd/node/multi_node.go
Comment (y0sher) [inline cmd/node/multi_node.go:161]: maybe get this from param or config ?

File: cmd/node/multi_node.go
Comment (y0sher) [inline cmd/node/multi_node.go:199]: I'd refactor this loop outside of this function

File: cmd/node/node.go
Comment (y0sher) [inline cmd/node/node.go]: old code ? remove it

File: cmd/node/node.go
Comment (y0sher) [inline cmd/node/node.go:586]: you could do this outside this func imo

File: collector/collector.go
Comment (y0sher) [inline collector/collector.go]: general note: it would be better if you created a named logger for the collector instead of using the logger.

File: collector/collector.go
Comment (y0sher) [inline collector/collector.go:101]: maybe this should be explained better

File: miner/block_builder.go
Comment (y0sher) [inline miner/block_builder.go]: we see released tick every layer this should be corresponding.. either debug or remote

File: timesync/clock.go
Comment (y0sher) [inline timesync/clock.go:176]: this log has no context

File: turbohare/hare.go
Comment (y0sher) [inline turbohare/hare.go:33]: nice log 👍

File: collector/collector.go
Comment (y0sher) [inline collector/collector.go:90]: you don't check the error for each subscription but only for the last

File: collector/collector.go
Comment (y0sher) [inline collector/collector.go]: you start listening before checking the error

File: cmd/node/app_test.go
Comment (noamnelke) [reply cmd/node/app_test.go:118]: The `go-spacemesh` repo depends on a specific version of the poet. Unit tests don't use the latest, this is a problem in automation tests only. The only reason this would break is if someone upgrades `go-spacemesh` before this PR is merged, and it will cause bors to block merging, since tests will fail.

File: timesync/clock.go
Comment (antonlerner) [reply timesync/clock.go:176]: this is good if you want to count number of subscribers

File: collector/collector.go
Comment (antonlerner) [reply collector/collector.go]: this is used only for debug and there should be only one collector per system, so naming this logger is not that important

File: cmd/node/node.go
Comment (antonlerner) [reply cmd/node/node.go:586]: I've refactored this so this function decided which hare to take according to parameters

File: cmd/node/multi_node.go
Comment (antonlerner) [reply cmd/node/multi_node.go:199]: what is the purpose?

File: cmd/node/multi_node.go
Comment (antonlerner) [reply cmd/node/multi_node.go]: dont you think this is clearer?

**Comment by antonlerner** [comment]
bors try

File: cmd/node/multi_node.go
Comment (y0sher) [reply cmd/node/multi_node.go]: not really. this format has a name. if you look it up (or read the const value) you'll see the format.

File: activation/activation.go
Comment (y0sher) [inline activation/activation.go]: deadbabe?

File: cmd/node/multi_node.go
Comment (y0sher) [reply cmd/node/multi_node.go:199]: the function's name is StartMultiNode.. this is not what you're doing in this loop

File: cmd/node/multi_node.go
Comment (antonlerner) [reply cmd/node/multi_node.go:161]: why would anybody want to change this?

File: activation/activation.go
Comment (y0sher) [inline activation/activation.go]: also deadbabe, why not something meaningful ?

File: cmd/node/node.go
Comment (y0sher) [reply cmd/node/node.go:586]: imo super hare should go outside so it is noticed, but it's not super important ;)

File: collector/collector.go
Comment (y0sher) [inline collector/collector.go]: now you don't really need this check because you've already check all errs.

**Comment by antonlerner** [comment]
bors merge

**Review by y0sher — APPROVED** [review]

**Comment by antonlerner** [comment]
bors merge

File: cmd/node/node.go
Comment (noamnelke) [inline cmd/node/node.go:3]: Hey man, can you explain this `import`? It causes our entire project to use `cgo` and breaks cross-compilation. I also don't think it's used anywhere...

**Pull request closed without merging**
